src/main/java/edu/harvard/iq/dataverse/export/ddi/DdiExportUtil.java

```diff
@@ -1610,4 +1632,47 @@ private static boolean checkParentElement(XMLStreamWriter xmlw, String elementNa
         return true;
     }
 
+    public static void datasetHtmlDDI(JsonObject datasetDtoAsJson, DatasetVersion version, OutputStream outputStream) throws XMLStreamException {
```

Comment:
How much effort would it be to write a test for this "datasetHtmlDDI" method?

Reply:
To write a test to see that conversion from xml to html is successful would not be difficult.  It can be added to DdiExportUtilTest. But to parse the resulting html file to see that the fields of xml converted properly to html would be quite cumbersome and time consuming.

Reply:
@lubitchv ok, if you'd like to write the easier test, please let us know if you'd like us to pull this out of QA until you're done. Or we can leave it in QA and tests can be added in a future pull request. It's up to you. 😄

Reply:
Ok, I will try to write the easier test on Monday. I think this PR can wait till I finish that test.

Reply:
@lubitchv sounds good. Please ping us when you're ready for us to move it back to code review. Have a nice weekend!

Reply:
Hey @lubitchv - thanks for the commit adding the test. Is this ready for another round of code review and then QA?

Reply:
@djbrooke Not yet, I need to check some things out. I will let you know when its ready.

Reply:
@djbrooke  It is now ready for code review and QA.

Reply:
Thanks @lubitchv! We'll take another look.